Doc/library/functions.rst

```diff
@@ -692,6 +692,12 @@ are always available.  They are listed here in alphabetical order.
    ``x.foobar``.  If the named attribute does not exist, *default* is returned if
    provided, otherwise :exc:`AttributeError` is raised.
 
+   .. note::
+
+      Since :ref:`private name mangling <private-name-mangling>` happens at
+      compilation time, private attributes are not found by :func:`getattr`
```

Comment:
I'm not happy with the use of the term ``private attribute`` here. It's also used in the context of attributes with single leading underscore. How about `attributes with two leading underscores`` instead?

Reply:
Oops you're right, I forgot that single leading underscores are hints for private attributes too

Reply:
FWIW the former form was also clear to me, since there was a reference to [the relevant piece of documentation](https://docs.python.org/3/reference/expressions.html#atom-identifiers) so the context was there.